### ISSUE TITLE
Local Radio overlay does not show in SteamVR

### ISSUE THREAD

**Issue by zflash** [Koenvh1/ets2-local-radio#151]
Not sure if this was implemented, but I saw some VR settings in the config about the overlay. The radio station overlay show up in the ATS / ETS2 window but not in the game itself. The only thing the VR variable in the config does, is flipping over the Overlay.

Do I need to do something else or has this never worked?

**Comment by Koenvh1** [comment]
So I don't have VR, so I cannot test it with it either. I once received a question from someone using VR, stating that the image was flipped, and if I could fix that. So I did.

**Comment by zflash** [comment]
Ah ok, that's understandable. Getting the overlay to work in VR requires some additional code then if you haven't done anything specific for it.

**Comment by sie1984** [comment]
Has this been looked into any further?

**Comment by Koenvh1** [comment]
Not really. Like I said, I don't have VR, so I can only apply transformations and scaling based on the reports from others who experience issues.

**Comment by sie1984** [comment]
Ah no worries its no big deal just be nice to see what station I was on whilst in VR. Just a separate question as I'm new to your app. Is there a way to stay on a radio station to prevent it changing in ATS when moving between states then change when I want it to?

**Comment by zflash** [comment]
I have it working now with a different approach. I open the radio browser window in a small frame. With CrewChief (or any other program), I can show this window in VR while riding. With a toggle on my buttonbox I can switch it on or of if i want. To get the keyboard mappings working via my buttonbox I use Joy2Key. Works great.
